src/flows/CostCalculator/CostCalculatorForm.tsx

```diff
@@ -54,9 +60,23 @@ export function CostCalculatorForm({
     } else {
       if (costCalculatorResults?.data) {
         await onSuccess?.(costCalculatorResults?.data);
+        // Reset the form completely
         if (shouldResetForm) {
```

Comment:
if `shouldResetForm` is true and `shouldResetFormFields` is defined, then `shouldResetForm` takes precedence. Is this the expected behaviour? If so, we should update the README.